### ISSUE TITLE
PostgreSQL: each time clicking on function "failed to find conversion function from unknown to regprocedure"

### ISSUE THREAD

**Issue by cschwaderer** [OmniDB/OmniDB#591]
This morning, I've updated from OmniDB 2.8 to 2.9.
Unfortunately, I've encountered an annoying behaviour: Each time, I click in the tree on a function, the message `failed to find conversion function from unknown to regprocedure` appears.

This happens on two different DB clusters on all databases I've tested. I've even tested a fresh DB: same behaviour.

The functions tested could be as simple and plain as
```

CREATE OR REPLACE FUNCTION public.my_dummy_function()
 RETURNS integer
 LANGUAGE SQL
 SECURITY DEFINER
AS $function$

SELECT 42

$function$
```

I'm using PostgreSQL 9.6 on Ubuntu 16.04.

**Comment by rafaelthca** [comment]
hi @cschwaderer,

Thanks for reporting. What OS are you using? OmniDB-app or OmniDB-server?

**Comment by cschwaderer** [comment]
OS is as mentioned Ubuntu 16.04. I'm using OmniDB-app only.

**Comment by rafaelthca** [comment]
Indeed, this was introduced in 2.9 when connecting to versions older than pg 10. 

Will fix it.

**Comment by wind39** [comment]
Hi @cschwaderer ,

Thanks for reporting this. I have implemented a fix, it will be published with the next release next week.

Regards,
William Ivanski
